portfolio/src/main/webapp/script.js

```diff
@@ -20,7 +20,22 @@ function off() {
 }
 // asynchronously fetch content from server
 async function getContent() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('content-container').innerText = quote;
+    const response = await fetch('/data');
+    const hardCodedContent = await response.json();
+  
+    const contentListElement = document.getElementById("content-container");
+    contentListElement.innerHTML = '';
+    contentListElement.appendChild(
```

Comment:
Do you need to append your `<li>..</li>` elements inside a ordered list (`<ol>`) or unordered list (`<ul>`) element?

Reply:
I used <ul> in index.html. That's what the walk-through used.

Reply:
(It was just filler text to test how the server sent back the response)